### PR TITLE
248 styling

### PR DESCRIPTION
**Pull request by dwindleduck**
Complete:
- added the total periods to the denominator of Number of Periods Included
- added static help icon images in three locations (should create a reusable component?)
- updated some styling to more closely match the design

Still needs attention:
- thousand commas on Whole-home UA BTU (use browser locale functionality or d3format)
- tailwind's border radius utilities  (rounded-md) not yet working

### CODE REVIEW

File: heat-stack/app/components/ui/heat/CaseSummaryComponents/AnalysisHeader.tsx [hunk 1]
```diff
@@ -58,40 +61,45 @@ export function AnalysisHeader({ usage_data }: { usage_data: UsageDataSchema}) {
...
 
 	return (
-		<div className="section-title">
-			<div className="item-group-title">Heat Load Analysis</div>
-			<div className="flex flex-row">
+		<div className="section-title mt-12">
+			<div className="flex flex-row gap-0.5 mb-4">
+				<div className="item-group-title font-semibold text-2xl">Heat Load Analysis</div>
+				{/* TODO: add help text here */}
+				<img src={HelpCircle} alt='help text'/>
```
Comment (plocket) [inline heat-stack/app/components/ui/heat/CaseSummaryComponents/AnalysisHeader.tsx:68]: I suggest commenting out the help circle until we add the help text. Or add the help text before merging this. I feel that's pretty important.

```suggestion
				{/* TODO: add help text here */}
				{/* <img src={HelpCircle} alt='help text'/> */}
```

File: heat-stack/app/components/ui/heat/CaseSummaryComponents/EnergyUseHistoryChart.tsx [hunk 2]
```diff
@@ -82,19 +83,31 @@ export function EnergyUseHistoryChart({ usage_data }: { usage_data: UsageDataSch
...
-					<TableHead>Whole-home UA (BTU/h-°F)</TableHead>
-					<TableHead>Override Default</TableHead>
-					{/* TODO: add help text */}
+				<TableRow className='text-xs text-muted-foreground bg-neutral-50'>
+					<TableHead className="text-center">#</TableHead>
+					<TableHead className='text-center'>
+						<div className="flex flex-row">
+							<div className='text-right'>Allowed Usage*</div>
+							{/* TODO: add help text */}
+							<img src={HelpCircle} alt='help text' className='pl-2'/>
```
Comment (plocket) [inline heat-stack/app/components/ui/heat/CaseSummaryComponents/EnergyUseHistoryChart.tsx:94]:
```suggestion
							{/* TODO: add help text */}
							{/* <img src={HelpCircle} alt='help text' className='pl-2'/> */}
```

File: heat-stack/app/components/ui/heat/CaseSummaryComponents/EnergyUseHistoryChart.tsx [hunk 3]
```diff
@@ -82,19 +83,31 @@ export function EnergyUseHistoryChart({ usage_data }: { usage_data: UsageDataSch
...
-					<TableHead>Days in Period</TableHead>
-					<TableHead>Gas Usage (therms)</TableHead>
-					<TableHead>Whole-home UA (BTU/h-°F)</TableHead>
-					<TableHead>Override Default</TableHead>
-					{/* TODO: add help text */}
+				<TableRow className='text-xs text-muted-foreground bg-neutral-50'>
+					<TableHead className="text-center">#</TableHead>
+					<TableHead className='text-center'>
+						<div className="flex flex-row">
+							<div className='text-right'>Allowed Usage*</div>
```
Comment (plocket) [inline heat-stack/app/components/ui/heat/CaseSummaryComponents/EnergyUseHistoryChart.tsx:92]: When we add the help circle and text, remove the asterix.

File: heat-stack/app/components/ui/heat/CaseSummaryComponents/EnergyUseHistoryChart.tsx [hunk 4]
```diff
@@ -82,19 +83,31 @@ export function EnergyUseHistoryChart({ usage_data }: { usage_data: UsageDataSch
...
+					<TableHead className='text-center'>Start Date</TableHead>
+					<TableHead className='text-center'>End Date</TableHead>
+					<TableHead className='text-center'>Days in Period</TableHead>
+					<TableHead className='text-center'>Gas Usage (therms)</TableHead>
+					<TableHead className='text-center'>Whole-home UA (BTU/h-°F)</TableHead>
+					<TableHead className='text-center'>
+						<div className="flex flex-row">
+							<div className='text-right'>Override Default</div>
+							{/* TODO: add help text */}
+							<img src={HelpCircle} alt='help text' className='pl-2'/>
```
Comment (plocket) [inline heat-stack/app/components/ui/heat/CaseSummaryComponents/EnergyUseHistoryChart.tsx:107]:
```suggestion
							{/* TODO: add help text */}
							{/* <img src={HelpCircle} alt='help text' className='pl-2'/> */}
```

**Review by plocket — COMMENTED** [review]
This is looking great! Thanks for the svgs! I think we should fix the import statements (though I'm not sure which are related to this PR). I also strongly feel we should either add the help text or comment out the help circle images for now. There's also one typo I saw. I'm leaving this as a comment, though, in case others feel differently.

**Comment by thadk** [comment]
image

**Comment by thadk** [comment]
Will fix import issues on another PR

**Pull request merged**